client/src/components/ApplicantsList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
+import { getAllProfiles } from "../api/profiles";
+
+const ApplicantsList = () => {
+  const [profiles, setProfiles] = useState([]);
+  const fetchProfiles = async () => {
+    const response = await getAllProfiles();
+    setProfiles(response);
+  };
+
+  useEffect(() => {
+    fetchProfiles();
+  }, []);
+
+  return profiles.map(profile => {
+    return (
+      <div class="ui two column grid raised segment">
+        <div class="twelve wide column">
+          <label for="job title" class="ui white left ribbon label">
+            Job Title
+          </label>
+          <div class="ui small segment">
+            <h2 class="ui small header">{profile.job_title}</h2>
+          </div>
+          <label for="bio" class="ui white left ribbon label">
+            Bio
+          </label>
+          <div class="ui small segment">
+            <p>{profile.bio}</p>
+          </div>
+        </div>
+        <div class="three wide column">
+          <div>
+            <Button
+              fluid
+              class="ui primary button"
```

Comment:
Looking at the documentation at https://react.semantic-ui.com/elements/button/#types-emphasis, a primary button is created using `<Button primary>View Profile</Button>`. What this shows is that we don't need to add any CSS classes, because they are styled like this internally in the `<Button>` component itself!

For adding a space between these buttons, we'd use CSS since it is a styling issue. It looks like Semantic UI doesn't have utility classes so what we'd do is add our own CSS class here which adds a bottom margin.